### ISSUE TITLE
Color Constrast Errors - Homepage

### ISSUE THREAD

**Issue by lfarrell** [samvera/hyrax#2094]
### Descriptive summary

 When checking for  contrast errors on the hyrax homepage using the WAVE accessibility toolbar, there are contrast errors for a number of links. Tagging @mtribone on this issue per @julesies 

### Expected behavior
Expect sufficient contrast ratio for all text 

### Actual behavior

![contrast](https://user-images.githubusercontent.com/436691/32335684-cbfbfdb4-bfc3-11e7-9711-d44ed1e6cc43.png)
### Steps to reproduce the behavior

1. Load the hyrax homepage and validate the page with the WAVE toolbar

**Comment by mtribone** [comment]
This one is tricky because the text is over an element that has transparency which is over a background image. The test is grabbing the color from the masthead because that is the closest thing that has a background color set–which is wrong.

I've approximated the image with transparency and tested it using http://www.brandwood.com/a11y/ and it mostly, kinda passes. Once you change the browser width or the background image, its light/dark value will change and it may pass or fail.

The best thing to do is to put a background color on the `<a>` element, so that the text will always have the proper contrast ration regardless of transparency and background image. You know, just a little block of _solid_ color behind home, about, help, contact us.

**Comment by mtribone** [comment]
![screen shot 2017-11-02 at 1 02 16 pm](https://user-images.githubusercontent.com/4163828/32339725-06ac8a5e-bfcf-11e7-8155-192880c87955.png)

**Comment by julesies** [comment]
@mtribone thanks! So for something like this, image which will be changed by any institution, is there anything to be done in hyrax?

**Comment by mtribone** [comment]
@julesies I recommend putting a solid color behind the links and leave the rest of it transparent to avoid the issue altogether. We could do that as a default, along with good defaults for banner colors, and it would be good for Hyrax. However, there is nothing that we can do about background images or foreground/background color if institutions make changes to defaults out o the box.

**Comment by julesies** [comment]
TY @mtribone! @vantuyls @mjgiarlo ☝️

**Comment by mtribone** [comment]
Kinda like the following by just switching the link background colors. Or we could add a span around the link and give it the background color with more of the transparency on the outside. Can get down right fancy if you wanna...

![screen shot 2017-11-02 at 1 58 58 pm](https://user-images.githubusercontent.com/4163828/32342180-09996f0a-bfd6-11e7-8a7a-b003ef454176.png)

**Comment by mjgiarlo** [comment]
@vantuyls @julesies 2.x series?
